playground/backend/internal/validators/java_validators.go

```diff
@@ -17,21 +17,43 @@ package validators
 
 import (
 	"beam.apache.org/playground/backend/internal/fs_tool"
+	"beam.apache.org/playground/backend/internal/logger"
+	"io/ioutil"
+	"strings"
 )
 
 const (
-	javaExtension = ".java"
+	javaExtension       = ".java"
+	javaUnitTestPattern = "@RunWith(JUnit4.class)"
 )
 
 // GetJavaValidators return validators methods that should be applied to Java code
+// The last validator should check that the code is unit tests or not
 func GetJavaValidators(filePath string) *[]Validator {
 	validatorArgs := make([]interface{}, 2)
 	validatorArgs[0] = filePath
 	validatorArgs[1] = javaExtension
 	pathCheckerValidator := Validator{
 		Validator: fs_tool.CheckPathIsValid,
 		Args:      validatorArgs,
+		Name:      "Valid path",
 	}
-	validators := []Validator{pathCheckerValidator}
+	unitTestValidator := Validator{
+		Validator: CheckIsUnitTests,
+		Args:      validatorArgs,
+		Name:      UnitTestValidatorName,
+	}
+	validators := []Validator{pathCheckerValidator, unitTestValidator}
 	return &validators
 }
+
+func CheckIsUnitTests(args ...interface{}) (bool, error) {
```

Comment:
Please add a comment here.

Reply:
I don't think there is a big need for a comment here, the function name is quite self-explanatory

Reply:
Do we have a test for this function?

Reply:
Done